src/components/tabs/tabs.module.scss

```diff
@@ -0,0 +1,63 @@
+@use "../../styles/abstracts/index" as *;
+
+.root {
```

Comment:
I guess here better to use `tabs` instead of `root`

Reply:
fix